charts/cluster-registry-client/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+{{- $isPercentage := regexMatch "^[0-9]+%$" (.Values.podDisruptionBudget.minAvailable | quote) }}
```

Comment:
The option is already there, this is to check if it is percentage or not so that we can validate that replicas is always greater than pdb minAvailable when the value is an integer and not percentage